### PR TITLE
Added add Feature

### PR DESCRIPTION
**Pull request by datobobokh**
Add Feature

### CODE REVIEW

File: controllers/expenses.controller.js [hunk 1]
```diff
@@ -1 +1,16 @@
-const {expenses} = require("../models");
\ No newline at end of file
+const {expenses} = require("../models");
+
+// post
+
+exports.add = (req, res) => {
+    if (!req.body["shop"]) {
```
Comment (Mike-Kirichenko) [inline controllers/expenses.controller.js]: add validation for price too

File: controllers/expenses.controller.js [hunk 2]
```diff
@@ -3,22 +3,33 @@ const {expenses} = require("../models");
...
+  const errorsArr = [];
     if (!req.body["shop"]) {
+      errorsArr.push("Text is required");
       res.status(422);
       res.send({answer: "Text is required"});
     }
     if (!req.body["spend"]){
+      errorsArr.push("Number is required");
       res.status(422);
       res.send({answer: "Number is required"});
```
Comment (M-kirichenko) [inline controllers/expenses.controller.js:15]: you send every error separately, that's not ok

Comment (Mike-Kirichenko) [inline controllers/expenses.controller.js]: you send every error separately, that's not ok

File: controllers/expenses.controller.js [hunk 3]
```diff
@@ -3,22 +3,33 @@ const {expenses} = require("../models");
...
     }
     if (req.body["spend"] < 0){
+      errorsArr.push("Number should be positive");
       res.status(422);
       res.send({answer: "Number should be positive"});
```
Comment (Mike-Kirichenko) [inline controllers/expenses.controller.js]: you send every error separately, that's not ok

File: controllers/expenses.controller.js [hunk 4]
```diff
@@ -3,22 +3,33 @@ const {expenses} = require("../models");
...
     }
     if (!req.body["date"]) {
+      errorsArr.push("Date is required");
       res.status(422);
       res.send({answer: "Date is required"});
```
Comment (Mike-Kirichenko) [inline controllers/expenses.controller.js]: you send every error separately, that's not ok

File: controllers/expenses.controller.js [hunk 5]
```diff
@@ -3,22 +3,33 @@ const {expenses} = require("../models");
...
     }
+    
+    if (errorsArr.length) {
+      res.status(422);
+      res.send({answer: errorsArr});
+    } 
+
     expenses.create(req.body)
```
Comment (Mike-Kirichenko) [inline controllers/expenses.controller.js:22]: always catch errors while executing query

File: controllers/expenses.controller.js (hunk 3 above)
Comment (Mike-Kirichenko) [reply controllers/expenses.controller.js]: add validation to spend which checks if it's number

File: controllers/expenses.controller.js (hunk 5 above)
Comment (Mike-Kirichenko) [reply controllers/expenses.controller.js:22]: if you manually pass data, then you got to validate it manually too

File: controllers/expenses.controller.js [hunk 6]
```diff
@@ -3,31 +3,21 @@ const {expenses} = require("../models");
 // post
 
 exports.add = (req, res) => {
+  const {shop} = req.body;
+  const {spend} = req.body;
```
Comment (Mike-Kirichenko) [inline controllers/expenses.controller.js]: destruct data in one line

**Pull request merged**